apps/emqx_bridge_azure_event_hub/test/emqx_bridge_azure_event_hub_producer_SUITE.erl

```diff
@@ -317,3 +317,28 @@ t_same_name_azure_kafka_bridges(AehConfig) ->
         end
     ),
     ok.
+
+t_create_with_bad_name(Config0) ->
```

Comment:
Not so sure about this, why test it here too? It's been reported for AEH, but the problem has nothing to do with it. Future people might start to wonder why this is tested here. Do we want to test that with every bridge?

Certainly we want to test whether the compat layer (i.e. using a v2 bridge works through the old `/bridges` API endpoint), but isn't that done elsewhere too?

Anyway, maybe leave a comment at least, and explain why this is here.